src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
```

Comment:
Similarly to the `Random` instance, consider creating a single instance of `ColorSupplier` as a class-level field to avoid creating a new instance every time the method is called.